tests/Gelf/Test/Transport/RetryTransportWrapperTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+
+namespace Gelf\Test\Transport;
+
+
+use Gelf\Message;
+use Gelf\TestCase;
+use Gelf\Transport\AbstractTransport;
+use Gelf\Transport\RetryTransportWrapper;
+use \PHPUnit_Framework_MockObject_MockObject as MockObject;
+
+class RetryTransportWrapperTest extends TestCase
+{
+    /**
+     * @var Message
+     */
+    private $message;
+
+    /**
+     * @var AbstractTransport|MockObject
+     */
+    private $transport;
+
+    /**
+     * @var RetryTransportWrapper
+     */
+    private $wrapper;
+
+    /**
+     * @var string
+     */
+    private $successValue;
```

Comment:
Since the value is static, you should use a constant here instead of a class property. Also, constants should be put on top of the class.

```suggestion
    private const SUCCESS_VALUE = "HTTP/1.1 202 Accepted\r\n\r\n";
```